modules/puppet_connect/plans/test_input_data.pp

```diff
@@ -0,0 +1,31 @@
+# @summary
+#   Tests that the provided Puppet Connect input data is complete, meaning that all consuming inventory targets are connectable.
+#
+# This plan should only be used as part of the copy-pastable "test input data"
+# workflow specified in the Puppet Connect docs.
+#
+# @param targets
+#   The inventory targets. This should never be set by the user. It is only here for compatibility with Bolt's rerun workflow.
```

Comment:
I don't know that we need to be quite so strong about this. Nothing bad will happen if you set it, it's just probably not what you want. Maybe "The set of targets to test. Usually this should be 'all', which is the default."

Reply:
Hmm, but it is why the parameter was added. I don't have strong feelings about making this more flexible, but I see no good reason to do that when this plan is meant for a very specialized workflow.

Reply:
It's more an issue of perception than making it more flexible. Having something that is _exposed_ to the user but is not _intended_ to be exposed to the user feels sloppy and I think it will make this feature feel less finished. But if we reframe it as this is a feature that is available but you probably don't need to use it, it doesn't feel like there's a hole.

Reply:
Ok, that reasoning makes sense. Updated.